### PR TITLE
Changed extends 'base.html' to extends 'generic/object_detail.html' for detail templates

### PR DESCRIPTION
**Pull request by briddo**
Fixes: #74

### CODE REVIEW

File: nautobot_ssot/templates/nautobot_ssot/data_source_target.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% extends 'base.html' %}
+{% extends 'generic/object_detail.html' %}
```
Comment (glennmatthews) [inline nautobot_ssot/templates/nautobot_ssot/data_source_target.html:1]: In addition to this change, need to review whether any or all of the contents of `block header` are now redundant, and if they can be removed to simplify the template.

Additionally, `block content` should be updated to only override `block content_[left|right|full_width]_page` as appropriate.

File: .gitignore [hunk 2]
```diff
@@ -189,6 +189,10 @@ $RECYCLE.BIN/
 *.lnk
 
 ### PyCharm ###
+
+# Jetbrains IDE configs
+.idea/
```
Comment (glennmatthews) [inline .gitignore:194]: Can presumably remove all of the individual `.idea/**/...` entries below since they're covered by this blanket entry now?

**Comment by gsnider2195** [comment]
This one is going to require some not insignificant refactoring. 

- `data_source_target.html` is used in a custom `View` (not `generic.ObjectView`) that maps to multiple custom job classes, not database models. 
- `sync_header.html` is being used as a base template for the main tab and a separate job result tab on the `Sync` object view

File: nautobot_ssot/templates/nautobot_ssot/data_source_target.html (hunk 1 above)
Comment (briddo) [reply nautobot_ssot/templates/nautobot_ssot/data_source_target.html:1]: After reviewing the `header` block, I think this customisation is necessary.

**Comment by snaselj** [comment]
Rebased on top of `develop`

**Comment by snaselj** [comment]
I'm creating a new PR instead of https://github.com/nautobot/nautobot-plugin-ssot/pull/76 as is seems to me not necessary to make these changes.

**Comment by bryanculver** [comment]
Replaced with #89

**Pull request closed without merging**